misc/experimental/rust/celix_bindings/build.rs

```diff
@@ -19,21 +19,37 @@
 
 extern crate bindgen;
```

Comment:
Why shall we need it? Seems unnecessary.

Reply:
Removing it is OK, while  removing other `extern crate`s will cause build errors, though all of them have been specified in the manifest's dependencies.